### ISSUE TITLE
Updating an empty AppLayout, after its initialization doesn't work

### ISSUE THREAD

**Issue by anasmi** [vaadin/vaadin-app-layout-flow#124]
**Version** : 14.1.3
**Scenario**: Based on the screen size, either use `Navbar` or `Drawer`
**Expected**: Layout is rendered correctly
**Actual**: elements are in the DOM, but not visible to the end user
**Code to reproduce**: 
```
import com.vaadin.flow.component.UI;
import com.vaadin.flow.component.applayout.AppLayout;
import com.vaadin.flow.component.applayout.DrawerToggle;
import com.vaadin.flow.component.html.Div;
import com.vaadin.flow.component.html.Image;
import com.vaadin.flow.component.tabs.Tab;
import com.vaadin.flow.component.tabs.Tabs;
import com.vaadin.flow.router.Route;
import com.vaadin.flow.theme.Theme;
import com.vaadin.flow.theme.lumo.Lumo;


@Theme(value = Lumo.class,variant=Lumo.LIGHT)
@Route("app-layout")
public class AppLayoutView extends AppLayout  {

    Image img = new Image("https://i.imgur.com/GPpnszs.png", "Vaadin Logo");
    Tabs tabs = new Tabs(new Tab("Home"), new Tab("About"));

    public AppLayoutView(){
        img.setHeight("44px");
        //Workaroung; adding an empty element
        addToNavbar(new Div());

        UI.getCurrent().getPage().retrieveExtendedClientDetails(event->{
            int width=event.getWindowInnerWidth();
            System.out.println("Details: screen width" + event.getWindowInnerWidth());
            if(width <1000) {
                //This is rendered empty
                System.out.println("Width less than 1000");
                setPrimarySection(AppLayout.Section.DRAWER);
                addToNavbar(new DrawerToggle(), img);
                addToDrawer(tabs);
            }
            else{
                //This is unresponsive
                addToNavbar(img, tabs);
            }
        });
    }
}


```
**Workaround**: add an empty element to the layout (Div in this case worked )

**Comment by tulioag** [comment]
I would guess this is an issue in the webcomponent. Resizing the browser window makes everything be shown again, without any network request.

**Comment by anasmi** [comment]
So could you transfer this issue?

**Comment by tulioag** [comment]
We can move it if necessary. I think it would be harder to reproduce without using flow.

**Comment by web-padawan** [comment]
Fixed by vaadin/vaadin-app-layout#146
